test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Link.cshtml

```diff
@@ -12,6 +12,9 @@
     <!-- Globbed link tag with existing file -->
     <link asp-href-include="**/site.css" rel="stylesheet" title="&lt;the title>" />
 
+    <!-- Globbed link tag with comma separated pattern -->
+    <link asp-href-include="~/*.css, ~/sub/*.css" rel="stylesheet" title="&lt;the title" />
```

Comment:
@Eilon we should support `~/` prefixes but is that something we should encourage i.e. highlight in this functional test web site?
@ajaybhargavb our coverage of comma-separated patterns is limited. Here or in `GlobbingUrlBuilderTest`, need to cover comma-separated patterns that do _not_ use the `~/` prefix.